hw/ip/otbn/data/otbn.hjson

```diff
@@ -201,6 +201,32 @@
       package: "keymgr_pkg"
     },
   ],
+  features: [
```

Comment:
Looking through the OTBN [documentation](https://opentitan.org/book/hw/ip/otbn/), one of the listed features is: "Full control-flow support with conditional branch and unconditional jump instructions, hardware loops, and hardware-managed call/return stacks."
Would it make sense to list some of it here as a separate feature?
I think that all of this is covered by `chip_sw_otbn_isatest`.

Reply:
I'd prefer not to put that as a separate feature, I want to avoid having a massive feature list here and if we're listing specific ISA features it becomes large quickly and I don't think we need different sival tests for all of them. What I'm focussing on here is the features that involve interfacing with other parts of the chip.